server/routerlicious/packages/services/src/redisClientManager.ts

```diff
@@ -16,13 +16,17 @@ export class ClientManager implements IClientManager {
     private readonly expire: any;
 
     constructor(
-        client: RedisClient,
+        client: Redis,
         private readonly expireAfterSeconds = 60 * 60 * 24,
         private readonly prefix = "client") {
-        this.addAsync = util.promisify(client.hmset.bind(client));
-        this.removeAsync = util.promisify(client.hdel.bind(client));
-        this.findAllAsync = util.promisify(client.hgetall.bind(client));
-        this.expire = util.promisify(client.expire.bind(client));
+        this.addAsync = client.hmset.bind(client);
+        this.removeAsync = client.hdel.bind(client);
+        this.findAllAsync = client.hgetall.bind(client);
+        this.expire = client.expire.bind(client);
```

Comment:
If these new libraries are already promisified, can we just use those directly?

Reply:
I went ahead and removed the class' custom fields and used the redis client directly.